src/main/java/games/strategy/engine/framework/startup/ui/PBEMSetupPanel.java

```diff
@@ -282,8 +282,9 @@ private void loadEmailSender(final GameData data) {
    *        the type of class
    * @return a IBean either loaded from the cache or created
    */
-  private static IBean findCachedOrCreateNew(final Class<? extends IBean> theClassType) {
-    IBean cached = LocalBeanCache.INSTANCE.getSerializable(theClassType.getCanonicalName());
+  private static <T extends IBean> T findCachedOrCreateNew(final Class<T> theClassType) {
+    @SuppressWarnings("unchecked")
+    T cached = (T) LocalBeanCache.INSTANCE.getSerializable(theClassType.getCanonicalName());
```

Comment:
I think you can get rid of the `@SuppressWarning` if you use the local `Class<T>` variable to cast:

```java
T cached = theClassType.cast(LocalBeanCache.INSTANCE.getSerializable(theClassType.getCanonicalName()));
```